src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -93,7 +103,7 @@ export const SortDropdownContainer = styled.div<{ isFiltersExpanded: boolean }>`
 
 const EmptyCollectionWrapper = styled.div`
   display: block;
-  textalign: center;
+  text-align: center;
```

Comment:
👀

Reply:
yeah that spooked me a little

need to get that dangling vscode plugin suggestions pr merged

Reply:
yikes. eslint plz